terminus-ui/select/testing/src/test-helpers.ts

```diff
@@ -96,3 +96,7 @@ export function getFilterInputElement(fixture: ComponentFixture<any>): HTMLInput
   return fixture.debugElement.query(By.css('.ts-select-panel__filter-input .c-input__text')).nativeElement as HTMLInputElement;
 }
 
+export function getValidationMessageElement(fixture: ComponentFixture<any>): HTMLInputElement {
```

Comment:
This is something we could leverage in other components! No changes, just noting it.

Reply:
@atlwendy, Shani and I chatted through this and for now you can just leave it as is. I'm going to open up a new ticket to create a top level testing endpoint (`@terminus/ui/testing`) to expose these more widely used utilities that cross components.